src/app/api/v1/admin/feature-flags/[flagId]/route.ts

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextRequest, NextResponse } from "next/server";
+import { getToken } from "next-auth/jwt";
+import { getFeatureFlagByName } from "../../../../../../db/tables/featureFlags";
+import { isAdmin } from "../../../../utils/auth";
+import appConstants from "../../../../../../appConstants";
+
+export async function GET(req: NextRequest) {
+  const token = await getToken({ req });
```

Comment:
I think Next-Auth's idea is that we use `getSession` to read session data, rather than directly inspecting the data inside the token. I'm not sure yet what extra checks they wrap around it, but it's probably safer to go with that.

Reply:
Using session now